src/sonic-yang-models/yang-models/sonic-lldp.yang

```diff
@@ -0,0 +1,160 @@
+module sonic-lldp {
+    namespace "http://github.com/Azure/sonic-lldp";
+    prefix slldp;
+    yang-version 1.1;
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC LLDP yang model";
+
+    revision 2020-04-16 {
+        description
+            "Initial revision.";
+    }
+
+    grouping lldp_mode_config {
+		leaf enabled {
+			type boolean;
+			description
+				"Enable/Disable LLDP";
+		}
+
+		leaf mode {
+			type enumeration {
+				enum RECEIVE;
+				enum TRANSMIT;
+			}
+			
+			description
+				"RX/TX mode for LLDP frames";
+		}
+    }
+
+    container sonic-lldp {
+        container LLDP {
+            list LLDP_LIST {
+                key "id";
+
+                leaf id {
+                    type enumeration {
+                        enum GLOBAL;
+                    }
+                    description
+                        "It's value is always GLOBAL";
+                }
+
+
+                leaf hello_time {
+                    type uint8 {
+                        range "5..254" {
+                            error-message "Invalid hello timer value.";
+                        }
+                    }
+                    units seconds;
+                    description
+                        "It is the time interval at which periodic hellos are
+                        exchanged. Default is 30 seconds";
+                }
+
+                leaf multiplier {
+                    type uint8 {
+                        range "1..10" {
+                            error-message "Invalid LLDP multiplier value.";
+                        }
+                    }
+                    description
+                        "This multiplier value is used to determine the timeout
+                        interval (i.e. hello-time x multiplier value) after
+                        which LLDP neighbor entry is deleted.";
+                }
+
+				leaf system_name {
+					type string;
+					description
+						"System administratively assigned name";
+				}
+
+				leaf system_description {
+					type string;
+					description
+						"System description";
+				}
+
+				leaf supp_mgmt_address_tlv {
+					type boolean;
+					description
```

Comment:
should we have config option to advertise v4/v6/both management capability.
can we check that

Reply:
Abhishek, i checked the LLDPd implementation currently there is no support to control advertising mgmt capability TLV for v4 and v6 separately, once that support is available we can probably make the changes to YANG, let me know your thoughts.